AEPCore/Sources/configuration/Event+Configuration.swift

```diff
@@ -24,6 +24,11 @@ extension Event {
         return data?[ConfigurationConstants.Keys.RETRIEVE_CONFIG] as? Bool ?? false
     }
 
+    /// Returns true if this `Event` is a clear configuration event, otherwise false
+    var isClearConfigEvent: Bool {
+        return data?[ConfigurationConstants.Keys.CLEAR_CONFIG] != nil
```

Comment:
may be better to check the value here, rather than just the existence of the property.  similar to `isGetConfigEvent` above?